package.json

```
@@ -48,28 +48,28 @@
         "debug": "~4.3.4",
         "etl": "^0.7.0",
         "express": "~4.18.1",
-        "express-fileupload": "^1.3.1",
-        "express-openapi": "^11.0.0",
+        "express-fileupload": "^1.4.0",
+        "express-openapi": "^11.0.1",
         "express-session": "^1.17.3",
         "http-errors": "~2.0.0",
         "inflected": "^2.1.0",
         "latest": "^0.2.0",
         "mime": "^3.0.0",
         "morgan": "~1.10.0",
-        "mssql": "^8.1.0",
+        "mssql": "^8.1.1",
         "mysql": "^2.18.1",
         "n3": "^1.16.2",
         "ndjson": "^2.0.0",
         "nodemon": "^2.0.16",
         "npm": "^8.10.0",
-        "passport": "^0.5.2",
+        "passport": "^0.6.0",
```

Cette màj casse l'authentification, je propose de la laisser à 0.5.2
```suggestion
        "passport": "^0.5.2",
```

---

Ou nous pouvons réparer l'erreur, il s'agit d'ajouter un callback à req.logout() dans le fichier api/v1/paths/auth/logout.js

---

oui, passons à la version 0.6 et ajoutons le callback nécessaire